auto-generated/vector-crypto/llvm-api-tests/vctz.c

```diff
@@ -1,12 +1,9 @@
 // REQUIRES: riscv-registered-target
-// RUN: %clang_cc1 -triple riscv64 -target-feature +v -target-feature +zvl512b \
+// RUN: %clang_cc1 -triple riscv64 -disable-O0-optnone \
 // RUN:   -target-feature +zvbb \
-// RUN:   -target-feature +zvbc \
-// RUN:   -target-feature +zvkg \
-// RUN:   -target-feature +zvkned \
-// RUN:   -target-feature +zvknhb \
-// RUN:   -target-feature +zvksed \
-// RUN:   -target-feature +zvksh -disable-O0-optnone \
+// RUN:   -target-feature +zve64x \
+// RUN:   -target-feature +zvl256b \
```

Comment:
Why does this require zvl256b?

Reply:
I think you are right, we don't need it.
The `zvl` constraint is generated by `EGW / LMUL` , but it is not applicable on `Zvbb` intrinsics, I will try if I can find a way to handle this.

Reply:
Fixed, thanks.